### PR TITLE
Document sendConsoleLogs and sendCoverageReports

### PR DESCRIPTION
**Pull request by deepikasl**
https://github.com/Shippable/docsv2/issues/784

### CODE REVIEW

File: sources/legacyFiles/int_notifications.md [hunk 1]
```diff
@@ -49,6 +49,17 @@ You have now configured your private IRC integration and can use this to send no
...
+```yaml
+notifications:
+    - integrationName: email
+      type: email
+      recipients:
+        - deepika@shippable.com
+      sendLogsInEmail: true
+```
+
 You can, however, add an Email account integration to receive notifications for monitored Docker images. To do this,
```
Comment (rageshkrishna) [inline sources/legacyFiles/int_notifications.md]: I think this whole section can be deleted. The lighthouse feature no longer exists, which is what this section is talking about (lines 63 - 69 in the new file)

File: sources/legacyFiles/int_notifications.md [hunk 2]
```diff
@@ -49,6 +49,17 @@ You have now configured your private IRC integration and can use this to send no
 
 Build Emails are always sent through our Shippable account and are configured through shippable.yml. At this time, you cannot customize the domain emails are sent from. To request this enhancement, please contact [customer support](mailto:support@shippable.com)
 
+To get console logs, coverage reports as attachments in build report to your email, add `sendLogsInEmail: true` in email integration in shippable.yml.
```
Comment (rageshkrishna) [inline sources/legacyFiles/int_notifications.md]: You can set `sendLogsInEmail: true` in your email notification configuration in shippable.yml to receive console logs and coverage reports as attachments in build emails.

**Comment by deepikasl** [comment]
@manishas can you please take a look at this PR.

**Comment by manishas** [comment]
I am ok with the docs PR. However, I am not sure why we didn't call the tag 'sendLogs'.... why make it so specific to email? This is a design question though.

**Comment by manishas** [comment]
Also, it's more than logs... looks like test and coverage reports are included.

**Comment by manishas** [comment]
@deepikasl did we skip sendTestReports?

**Comment by deepikasl** [comment]
@manishas Right now, we dont have any route to download, test reports. We will get back to this, once we add that route.

**Pull request merged**